JustEat.StatsD.Common.props

```diff
@@ -1,4 +1,5 @@
 <Project>
+ <Import Project="version.props" />
```

Comment:
While you're at it, if you rename this to `Directory.Build.props`, it won't need manually importing by name in the `csproj` files.

Reply:
Done